Sources/SwiftMetricsDash/SwiftMetricsDash.swift

```diff
@@ -98,31 +104,23 @@ class SwiftMetricsService: WebSocketService {
     }
 
 
-
     func sendCPU(cpu: CPUData) {
-        let cpuLine = JSON(["topic":"cpu", "payload":["time":"\(cpu.timeOfSample)","process":"\(cpu.percentUsedByApplication)","system":"\(cpu.percentUsedBySystem)"]])
+        let cpuDashData = DashData(topic: "cpu", payload: cpu)
+        let data = try! encoder.encode(cpuDashData)
 
         for (_,connection) in connections {
-            if let messageToSend = cpuLine.rawString() {
-                connection.send(message: messageToSend)
-            }
+          connection.send(message: String(data: data, encoding: .utf8)!)
```

Comment:
Suggest getting rid of the forced unpacking by doing the following
``` swift
if let data = encoder.encode(cpuDashData) {
   for (_,connection) in connections {
      connection.send(message: String(data: data, encoding: .utf8))
  }
}

Reply:
The swift compiler doesn't like your version because encode throws:
```
error: initializer for conditional binding must have Optional type, not 'Data'
        if let data = encoder.encode(cpuDashData) {
           ^
/home/sian/SwiftMetrics/Sources/SwiftMetricsDash/SwiftMetricsDash.swift:110:23: error: call can throw, but it is not marked with 'try' and the error is not handled
        if let data = encoder.encode(cpuDashData) {
                      ^
```
We can do 
```swift
    if let data = try? encoder.encode(cpuDashData) {
        for (_,connection) in connections {
            connection.send(message: String(data: data, encoding: .utf8)!)
        }
    }
```
which is slightly neater but doesn't get rid of the forced unpack.